### PR TITLE
Mm 20897 add ability to mute categories

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://mattermost.atlassian.net/browse/MM-20897

Updated:
- User's Guide > Organizing Conversations > Experimental: Channel Organization Features
   - Added mute/unmute and rename custom category functionality details

Also as part of this PR:
- User's Guide > Organizing Conversations > Channel Types
   - Updated "Direct Messages" bullet point to match heading as "Direct Messages and Group Messages"
- User's Guide > Organizing Conversations > Managing Channels
   - Updated introduction to include the ability to mute/unmute channels & mark as a favorite
- User's Guide > Organizing Conversations > Managing Channels > Favoriting a Channel
   - Reduced the size of both images

### CODE REVIEW

File: source/help/getting-started/organizing-conversations.rst [hunk 1]
```diff
@@ -137,13 +137,30 @@ To mark a channel as a favorite, open the channel:
...
 **On mobile:** Open the dropdown list by the channel name, and select **Favorite**. To remove a channel from the **Favorites** list, select **Favorite** again.
 
+Muting a channel
+~~~~~~~~~~~~~~~~
+
+Select the channel name at the top of the center pane to access the drop-down menu, then select **Mute Channel**. 
+
+Once a channel is muted:
+
+- Email and push notifications are disabled.
```
Comment (esethna) [inline source/help/getting-started/organizing-conversations.rst:158]: Also desktop notifications

File: source/help/getting-started/organizing-conversations.rst [hunk 2]
```diff
@@ -154,7 +171,11 @@ When configured by the System Admin, users can enable the features in **Account
 
 **Collapsible custom categories**: Create custom categories in the sidebar to group channels together for easier navigation (e.g. “Design” or “Marketing”). Collapse categories to display only unread channels and reduce unnecessary scrolling.
 
-**Drag and drop channels and categories**: Drag channels between or within categories, or drag to reorder entire categories to prioritize important conversations.
+**Drag and drop channels and categories**: Drag channels between or within custom categories, or drag to reorder entire categories to prioritize important conversations.
+
+**Rename custom categories**: Customize name of categories.
+
+**Mute and unmute custom categories**: Mute and unmute all channels in a custom category.
```
Comment (esethna) [inline source/help/getting-started/organizing-conversations.rst:178]: You can also mute a category, then choose to unmute select channels, or vise versa

File: source/help/getting-started/organizing-conversations.rst [hunk 3]
```diff
@@ -154,7 +171,11 @@ When configured by the System Admin, users can enable the features in **Account
 
 **Collapsible custom categories**: Create custom categories in the sidebar to group channels together for easier navigation (e.g. “Design” or “Marketing”). Collapse categories to display only unread channels and reduce unnecessary scrolling.
 
-**Drag and drop channels and categories**: Drag channels between or within categories, or drag to reorder entire categories to prioritize important conversations.
+**Drag and drop channels and categories**: Drag channels between or within custom categories, or drag to reorder entire categories to prioritize important conversations.
+
+**Rename custom categories**: Customize name of categories.
```
Comment (esethna) [inline source/help/getting-started/organizing-conversations.rst:176]: I wouldn't consider renaming at the same level of importance as the other things in this list. Renaming kind of goes without saying. This is more of a features highlight list in my mind

**Review by esethna — CHANGES_REQUESTED** [review]

File: source/help/getting-started/organizing-conversations.rst (hunk 1 above)
Comment (cwarnermm) [reply source/help/getting-started/organizing-conversations.rst:158]: Great addition! Thanks!

File: source/help/getting-started/organizing-conversations.rst (hunk 3 above)
Comment (cwarnermm) [reply source/help/getting-started/organizing-conversations.rst:176]: Sounds good. Thanks! I've removed the bullet point for renaming.

File: source/help/getting-started/organizing-conversations.rst (hunk 2 above)
Comment (cwarnermm) [reply source/help/getting-started/organizing-conversations.rst:178]: Thanks!

**Review by esethna — APPROVED** [review]

**Comment by amyblais** [comment]
@cwarnermm Can you help resolve the merge conflict?

**Comment by cwarnermm** [comment]
@amyblais The merge conflict has been resolved. Thanks!

**Pull request merged**